CMakeLists.txt

```diff
@@ -4,16 +4,17 @@ PROJECT(COCAINE-PLUGINS)
 
 SET(CMAKE_INSTALL_RPATH_USE_LINK_PATH TRUE)
 
-OPTION(CACHE "Build LRU Cache plugin" ON)
-OPTION(CHRONO "Build Chrono Service plugin" ON)
-OPTION(DOCKER "Build Docker Isolation plugin" ON)
-OPTION(ELASTICSEARCH "Build ElasticSearch Service plugin" ON)
-OPTION(IPVS "Build IPVS Gateway Service plugin" ON)
+OPTION(CACHE "Build LRU Cache plugin" OFF)
```

Comment:
You can use something like http://pastebin.com/uZxZRV8R this script while you work on a single plugin instead of editing CMakeLists.txt